### PR TITLE
test(cli): add test for delete-project extension

### PR DESCRIPTION
**Pull request by watilde**
*Description of changes:*
Increase test coverage of amplify-cli package.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by watilde** [comment]
I'd include tests for remove-env-from-cloud.ts and path-manager.ts into this PR for Codecov.

File: packages/amplify-cli/src/__tests__/extensions/amplify-helpers/delete-project.test.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      deleteS3: true,
+      deleteAmplifyApp: true,
+    };
+    const result = await getConfirmation(context_stub);
+    expect(result).toStrictEqual(expected);
+  });
+});
+
+describe('deleteProject', () => {
+  it('should delete app', async () => {
```
Comment (edwardfoyle) [inline packages/amplify-cli/src/__tests__/extensions/amplify-helpers/delete-project.test.ts:75]: nit would be more descriptive to name this test "should delete app when force flag is set"

**Review by edwardfoyle — APPROVED** [review]
one nit, but LGTM

**Comment by edwardfoyle** [comment]
just saw this is marked as WIP. Feel free to @ me when it's ready and I can take another look.

In the future, you can create a draft PR if it's not ready to be reviewed yet.

File: packages/amplify-cli/src/__tests__/extensions/amplify-helpers/delete-project.test.ts (hunk 1 above)
Comment (watilde) [reply packages/amplify-cli/src/__tests__/extensions/amplify-helpers/delete-project.test.ts:75]: Updated the description, thank you!

**Comment by watilde** [comment]
@edwardfoyle Thank you for comment! This PR is ready to ship and I'll add more tests related to the test code I put into this PR separately.

**Review by cjihrig — APPROVED** [review]

**Pull request merged**